packages/transaction-controller/src/TransactionController.ts

```diff
@@ -789,8 +789,10 @@ export class TransactionController extends BaseController<
    *
    * @param ignoreNetwork - Determines whether to wipe all transactions, or just those on the
    * current network. If `true`, all transactions are wiped.
+   * @param address - If specified, only transactions originating from this address will be
+   * wiped on current network.
    */
-  wipeTransactions(ignoreNetwork?: boolean) {
+  wipeTransactions(ignoreNetwork?: boolean, address?: string) {
     /* istanbul ignore next */
     if (ignoreNetwork) {
```

Comment:
Do we have to consider the `address` filter here also, and only return an empty array if both filter arguments are not specified?

Reply:
Done